config/plugins/FortressWars/settings.yml

```diff
@@ -419,6 +419,14 @@ settings:
           padding: false
           lines:
             - content: "&d[TIP] &7During build phase, you can sneak and place to build blocks in a &a3x3 &7pattern!"
+        - message:
+          padding: false
+          lines:
+            - content: "&d[TIP] &7Coordinate with your teammates more easily using the &a/teamchat &7command!"
```

Comment:
Please rephrase to "Easily coordinate with your teammates using the /teamchat command!